configs/common/Options.py

```diff
@@ -269,6 +269,8 @@ def addCommonOptions(parser, configure_xiangshan=False):
                         help="enable bp database for specified subdatabase, "
                         "basic branch trace is enabled by default even without specifying, "
                         "available subdatabase: basic, tage, ras, loop")
+    parser.add_argument("--disable-sc", default=False, action="store_true",
+                        help="enable sc (only for ftb branch predictor)")
```

Comment:
maybe we should use --enable-sc, not --disable-sc, help= "enable sc(for tage predictor)", this code writing style is more unified.